app/assets/stylesheets/pages/track-page.scss

```diff
@@ -157,24 +158,23 @@ body.controller-my-tracks.action-show {
             background-image:image-url("code-sample-frame.png");
             background-size:contain;
             background-repeat:no-repeat;
-            padding:30px 10px;
+            padding:28px 6px;
             height:200px;
             width: 258px;
 
             pre {
-                background:#fafafa;
```

Comment:
This file shouldn't need to set background colors. It should be done in one central place in the solution file.